pkg/deviceutils/device-utils_linux.go

```diff
@@ -24,9 +24,16 @@ import (
 	"k8s.io/mount-utils"
 )
 
+// DisableDevice asks the kernel to disable a device via /sys.
+// NB: this can be dangerous to use. Once a device is disabled, it's unusable, and can't be enabled
+// unless the serial number is known. But the serial number cannot be read from the device as it's
+// disabled. If a device is disabled in NodeUnstage, and then NodeStage is called without a
+// NodeUnpublish & publish sequence, the disabled state of the device will cause NodeStage to fail.
+// So this can only be used if we track the serial numbers of disabled devices in a persisent way
+// that survives driver restarts.
 func (_ *deviceUtils) DisableDevice(devicePath string) error {
```

Comment:
Why not just remove the entire function from deviceUtils interface? If we leave it in, this function isn't used and the code path isn't even being tested. If we really want to add it back in later, we can always look back at this PR revision to understand what was attempted before.

Reply:
I snuck in the newline fix. Can we submit this to get it in the repo, then I'll delete it in a followup CL? I also like the comment being attached to the method as it's hard to link up the PR to the code.

Reply:
Sure, SG